backend/routers/medical_documentation.py

```diff
@@ -0,0 +1,57 @@
+from fastapi import APIRouter, Depends, Header, HTTPException
+from typing import Annotated
+from models.database import SessionLocal
+import models.medical_documentation as models
+import schemas.medical_documentation as schemas
+from sqlalchemy.orm import Session, joinedload
+
+router = APIRouter()
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+async def create_medical_documentation_if_not_exists(db: Session, user_id : int = 1) -> models.MedicalDocumentation:
+    medical_documentation = db.query(models.MedicalDocumentation).filter(models.MedicalDocumentation.patient_id == user_id).first()
+    if medical_documentation is None:
+        medical_documentation = models.MedicalDocumentation(patient_id=user_id)
+        db.add(medical_documentation)
+        db.commit()
+        print("DOne")
+        
+    return medical_documentation
+    
+@router.get("/medical_documentation")
+async def get_medical_documentation(user_id: int = Header(1), db: Session = Depends(get_db)):
+    if db.query(models.MedicalDocumentation).filter(models.MedicalDocumentation.patient_id == user_id).first() is None:
```

Comment:
If the query is empty we get internal server error, so possibly this expression raises an exception when no item found